playbooks/viya-mmsu/viya-services-start-restore.yml

```diff
@@ -0,0 +1,26 @@
+  ####################################################################
+  #### viya-services-start-restore.yml                            ####
+  ####################################################################
+  #### Author: SAS Institute Inc.                                 ####
+  ####################################################################
+  #
+  # Copyright (c) 2025, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+  # SPDX-License-Identifier: Apache-2.0
+  #
+---
+  - import_playbook: ../common/handle_hostgroup_hyphens.yml
+    tags:
+      - always
+
+  - import_playbook: viya-services-start-prereq.yml
+    tags:
+      - always
+
+  - hosts: sas_all
+    become: yes
+    vars_files:
+      - viya-services-start-restore-vars.yml
+    tasks:
+      - name: Starting minimal services before a Restore operation
+        script: viya-svs.sh start {{ item }}
+        loop: "{{ restore_minimal_services }}"
```

Comment:
this will start them one at a time, and [wait until each is finished starting](https://github.com/sassoftware/viya-ark/blob/d821dfc385e6df0db98b11856c7e5b10c8220ab8/playbooks/viya-mmsu/viya-svs.sh#L92-L109) before starting the next. Instead it might be more efficient to send the entire list to `viya-svs.sh start` command, which supports being given 1 or more service names to start.

Reply:
I did try out proposed changes to pass entire list of services to the script and it does function the same.  I haven't bench-marked any performance improvements.   I haven't pushed the changes yet; here is what they are though:
```
+++ b/playbooks/viya-mmsu/viya-services-start-restore.yml
@@ -22,5 +22,4 @@
       - viya-services-start-restore-vars.yml
     tasks:
       - name: Starting minimal services before a Restore operation
-        script: viya-svs.sh start {{ item }}
-        loop: "{{ restore_minimal_services }}"
+        script: viya-svs.sh start "{{ restore_minimal_services | join(' ') }}"
```
Do you want me to push these changes @erharb ?

Reply:
yep!